concourse/pipelines/guest-package-build.yaml

```diff
@@ -1692,6 +2175,17 @@ resources:
   source:
     uri: https://github.com/GoogleCloudPlatform/compute-image-tools.git
     branch: master
+- name: compute-image-windows
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/compute-image-windows.git
+    branch: master
+- name: compute-image-windows-tag
+  type: github-release
+  source:
+    owner: GoogleCloudPlatform
+    repository: compute-image-windows
+    access_token: ((github-token.token))
```

Comment:
fyi we'll need to grant the robot permission to this repo so it can tag it.

Reply:
Done and accepted the invitation.